rplugin/python3/deoplete/__init__.py

```diff
@@ -41,6 +42,12 @@ def init_python(self, args):
 
     @neovim.rpc_export('deoplete_auto_completion_begin')
     def completion_begin(self, context):
+        if not logger.init:
```

Comment:
Can not debugging in `deoplete_manual_completion_begin`.
So, I think better way is that method moves to into `self.__deoplete.completion_begin(context)`

https://github.com/tweekmonster/deoplete.nvim/blob/logging/rplugin%2Fpython3%2Fdeoplete%2Fdeoplete.py#L57